### DISCUSSION TITLE
React does not recognize the `offsetX` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `offsetx` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

### DISCUSSION THREAD

**Discussion by ybf970928** [ant-design/ant-design#49120] [Q&A]
自定义Select 下拉框内容, dropdownRender返回一个table组件的话, 会提示React does not recognize the `offsetX` prop on a DOM

<img width="1434" alt="image" src="https://github.com/ant-design/ant-design/assets/63764959/59f6fa06-d75f-4c89-ba19-8c5f5bf8bbff">

**Comment by zhixiangyao** [comment]
+1
